bubuku/cli.py

```diff
@@ -100,6 +102,91 @@ def swap_partitions(threshold: int):
         RemoteCommandExecutorCheck.register_fatboy_slim(zookeeper, threshold_kb=threshold)
 
 
+@cli.group(name='actions', help='Work with running actions')
```

Comment:
@v-stepanov I don't fully get the question. It's group description if you run `bubuku-cli actions`